python/lsst/ip/diffim/psfMatch.py

```diff
@@ -1037,7 +1037,7 @@ def _solve(self, kernelCellSet, basisList, returnOnExcept=False):
 
         except pexExcept.Exception as e:
             pexLog.Trace(self.log.getName()+"._solve", 1, "ERROR: Unable to calculate psf matching kernel")
-            pexLog.Trace(self.log.getName()+"._solve", 2, e.args[0].what())
+            pexLog.Trace(self.log.getName()+"._solve", 2, e.args[0])
```

Comment:
Why not just e instead of e.args[0]? Can you be sure the other args are not interesting?